modules/arcspanRtdProvider.js

```diff
@@ -0,0 +1,71 @@
+import { submodule } from '../src/hook.js';
+import { mergeDeep } from '../src/utils.js';
+
+/** @type {string} */
+const MODULE_NAME = 'realTimeData';
+const SUBMODULE_NAME = 'arcspan';
+
+/** @type {RtdSubmodule} */
+export const arcspanSubmodule = {
+  name: SUBMODULE_NAME,
+  init: init,
+  getBidRequestData: alterBidRequests,
+};
+
+function init(config, userConsent) {
+  if (typeof config.params.silo === 'undefined') {
+    return false;
+  }
+  var _s = document.createElement('script');
```

Comment:
please use our methods to insert scripts

Reply:
also can you check for  window.arcobj1 before appending the script and recommend that it is deployed that way in your doc, you'll get better coverage that way. also, you'll want to make the location of the script configurable and fallback to a place that is locked to a version if you decide you do want prebid sourcing your location

Reply:
Thanks for the quick review, @patmmccann! I implemented the insert script method & the check, but I'm not sure I understand the 2nd part of your last comment. Do you mean we would have to make the location configurable and implement the fallback if/when we decide to have prebid source our location? Or do you mean we should make the location configurable and implement the fallback now, in preparation for if/when we decide to have prebid source our location?